src/Microsoft.Security.Utilities.Benchmarks/SecretMaskerDetectionBenchmarks.cs

```diff
@@ -91,8 +91,8 @@ private string GeneratePrefix(int length, bool random)
 
             if (random)
             {
-                var byteCount = Base64CharsToBytes(length);
-                var data = new byte[byteCount];
+                int byteCount = Base64CharsToBytes(length);
+                byte[] data = new byte[byteCount];
```

Comment:
> byte[]  [](http://example.com/codeflow?start=15&length=8)

var is fine here. why doesn't the compiler see this type as apparent?

Reply:
I believe it's because it has a separate knob for built-in types. And my choices are built-in = var or built-in = explicit and I don't know how to say built-in = defer to other knob. :(

Reply:
ok, i suppose setting `csharp_style_var_for_built_in_types` results in wild, perhaps objectionable code churn? i just want any `new` utilization to reduce to a var decl. i can live with what we have though!

Reply:
https://github.com/dotnet/roslyn/issues/26531 is related so we're not the only ones picking on this.

Reply:
oh hmm, while the issue is still open the last comment suggests there's some magic we can use. Trying.

Reply:
Nevermind, misread, that's a proposal, not implemented. I'll put a thumbs up.